src/corewlan.cs

```diff
@@ -544,4 +544,34 @@ interface CWNetworkProfile : NSCoding, NSSecureCoding, NSCopying, NSMutableCopyi
 	interface CWMutableNetworkProfile : NSCoding, NSSecureCoding, NSCopying, NSMutableCopying  
 	{
 	}
+
+	[BaseType (typeof (NSObject))]
```

Comment:
@spouliot The header lists it:

/*!
 * @method
 *
 * @abstract
 * Initializes a CWWiFiClient object.
 */
- (nullable CWWiFiClient *)init NS_AVAILABLE_MAC(10_10);